livestream-compose-sample/src/main/java/io/getstream/livestream/compose/streams/CameraLiveStream.kt

```diff
@@ -46,49 +54,81 @@ fun CameraLiveStream(
     val context = LocalContext.current
     val cameraProviderFuture = remember { ProcessCameraProvider.getInstance(context) }
 
-    LaunchedEffect(Unit) {
-        listViewModel.start()
-    }
-
-    ChatTheme {
-        Box(modifier = Modifier.fillMaxSize()) {
-            AndroidView(
-                factory = { ctx ->
-                    val preview = PreviewView(ctx)
-                    val executor = ContextCompat.getMainExecutor(ctx)
-                    cameraProviderFuture.addListener({
+    Box(modifier = Modifier.fillMaxSize()) {
+        AndroidView(
+            factory = { ctx ->
+                val preview = PreviewView(ctx)
+                val executor = ContextCompat.getMainExecutor(ctx)
+                cameraProviderFuture.addListener(
+                    {
                         val cameraProvider = cameraProviderFuture.get()
                         bindPreview(
                             lifecycleOwner,
                             preview,
                             cameraProvider
                         )
-                    }, executor)
-                    preview
-                },
-                modifier = Modifier.fillMaxSize(),
-            )
-            CommentsBox(
-                modifier = Modifier
-                    .background(
-                        brush = Brush.verticalGradient(
-                            listOf(Color.Transparent, Color.Black),
-                            0f,
-                            1050f,
+                    },
+                    executor
+                )
+                preview
+            },
+            modifier = Modifier.fillMaxSize()
+        )
+        //Gradient overlay
+        Spacer(
+            Modifier
+                .fillMaxWidth()
+                .fillMaxHeight(0.6f)
+                .align(Alignment.BottomCenter)
+                .background(
+                    brush = Brush.verticalGradient(
+                        colors = listOf(
+                            Color.Transparent,
+                            ChatTheme.colors.appBackground
                         )
                     )
-                    .align(Alignment.BottomCenter),
-                composerViewModel,
-                listViewModel
+                )
+        )
+        Column(
+            modifier = Modifier
+                .align(Alignment.BottomCenter)
+                .fillMaxHeight(0.5f)
+        ) {
+            MessageList(
+                modifier = Modifier
+                    .weight(1f),
+                viewModel = listViewModel,
+                itemContent = {
+                    LiveStreamComment(messageItem = it)
+                },
+                emptyContent = {
+                    // we hide default EmptyView from SDK ,
+                    // as we have a transparent scrim background for the video playing
+                    // in the background of our message list
+
+                    Spacer(
```

Comment:
I don't understand this completely. Why do we have the empty content as a spacer instead of using modifiers for alignment? Or a Scaffold that has a bottom bar?

Reply:
i need sample code how to fix this 

by not adding `empty` space 
```
Column(
            modifier = Modifier
                .align(Alignment.BottomCenter)
                .fillMaxHeight(0.5f)
        ) {
            MessageList(
                modifier = Modifier
                    .weight(1f),
                viewModel = listViewModel,
                itemContent = {
                    LiveStreamComment(messageItem = it)
                },
                emptyContent = {
                    // we hide default EmptyView from SDK ,
                    // as we have a transparent scrim background for the video playing
                    // in the background of our message list

                }
            )
            LivestreamComposer(
                channelId = channelId,
                composerViewModel = composerViewModel
            )
        }
 ```
 
 this is result: 
 <img src="https://user-images.githubusercontent.com/1392571/129479699-fa8ffe39-65e0-4439-b503-711f9565ad81.png" width=250>
 
 and by adding alignment to `last` element on column needs H alignment rather than V alignement 
 
 I dont fully got this working
 
 please advice